Documentation/troubleshooting.md

```diff
@@ -136,3 +136,11 @@ spec:
 ```
 
 The incorrect example will give an error along these lines `spec.endpoints.port in body must be of type string: "integer"`
+
+#### Prometheus/Alertmanager pods stuck in terminating loop with healthy start up logs
+
+Hint: Check that only one operator is running on the cluster! Having more than one operator wil lead to such beahaviour. To check run:
+
+`kubeclt get pods --all-namespaces | grep operator` 
```

Comment:
kubectl*

Reply:
You could filter this further using labels etc. There could be a lot of different operators, prometheus or otherwise running on a cluster.

Reply:
@PhilipGough Correct. I didn't check in my case if both operators had same labels. But I think running this ` k get pods -l app=kube-prometheus-stack-operator` would be able to catch if there is more than one prom operator running, correct?

Reply:
Only if it was deployed via the kube-prometheus stack :) It could be deployed in many different ways and I am not sure you will be able to accurately predict the labels.

I think it will be enough to say something like `Check for additionally deployed Prometheus operators`